exp-timing/femto.py

```diff
@@ -5,56 +5,56 @@
 import watchdog
 from psp.Pv import Pv
 import sys
-import random  # random number generator for secondary calibration
-from scipy.optimize import leastsq # for secondary calibration
+import random  # Used for random number generation during secondary calibration.
+from scipy.optimize import leastsq #Used in secondary calibration.
 
-class PVS():   # creates pvs
+class PVS():   # This class sets up all of the PVs used by the script. 
     def __init__(self, nx='NULL'):
-        self.version = 'Watchdog 141126a' # version string
-        self.name = nx
-        print(self.name)
-        namelist = set()
-        self.pvlist = dict()  # will hold pvs with names
-        matlab_list = dict() # list of matlab style pvs, tuples of matlab number offset and pv description
-        matlab_pv_base = dict() # header matlab pvs like SIOC:SYS0:ML00:OA....
-        matlab_pv_offset = dict() # start number for pvs
-        matlab_pv_digits = dict()  # number of digits for each pv
-        matlab_use = dict() # set to true if this matlab pv shoudl be used even if a ioc pv exists defined below.
+        self.version = 'Watchdog 141126a' #Version string
+        self.name = nx # Sets the hutch name.
+        print(self.name) # Prints the hutch name in log file.
+        namelist = set() # Sets up the namelist that will check the hutch name passed in by the IOC against the list of hutches femto.py is configured to run. 
+        self.pvlist = dict()  # Creates a list of all the PVs used by the script, excluding the notepad or "Matlab" PVs.  
+        matlab_list = dict() # List of the notepad or "Matlab" PVs, tuples of matlab number offset and pv description.
+        matlab_pv_base = dict() # Header Matlab PVs like SIOC:SYS0:ML00:OA. 
+        matlab_pv_offset = dict() # Start number for notepad PVs. A value of 1 is used of most hutches.
+        matlab_pv_digits = dict() # Number of digits for each notepad PV.
+        matlab_use = dict() # This value could be set to true to use a notepad PV instead of an IOC PV. This feature is not used in any of the lockers, though, so it is obsolete. 
         for n in range(0,20):
-            matlab_use[n] = True  # initize to always use matlab, override to use epics pvs.
-        counter_base = dict()  # holds counter name
-        freq_counter = dict() # frequencycounter name
-        dev_base = dict() # used to generate other device names (mostly future)
-        phase_motor = dict()
-        laser_trigger = dict()
-        trig_in_ticks = dict() # 1 if trigger units are ticks (1/119MHz), 0 if in nanoseconds( kludge for multi systems)
-        reverse_counter = dict()  # 1 if the laser starts the counter, trigger stops.        
-        error_pv_name = dict()    
-        use_secondary_calibration = dict() # use a scope or other device
+            matlab_use[n] = True  # Initize to always use notepad PV by default, override to use IOC PVs.
```

Comment:
# use notepad PV by default

What is it set to for the hutches? Always true, always false? Maybe we don't need this as an option.